sql-cli/sql_cli/__main__.py

```diff
@@ -150,19 +179,24 @@ def run(
         help="whether to explicitly generate the tasks in your SQL CLI DAG",
         show_default=True,
     ),
-    verbose: bool = typer.Option(False, help="Whether to show airflow logs", show_default=True),
+    verbose: bool = typer.Option(False, help="Whether to show verbose output", show_default=True),
 ) -> None:
     from airflow.utils.state import State
 
     from sql_cli.dag_runner import run_dag
     from sql_cli.project import Project
     from sql_cli.utils.airflow import get_dag
 
+    set_verbose_mode(verbose)
```

Comment:
Don't we also need to add `set_verbose_mode(verbose)` to the init command?

Reply:
By the time this PR has been open, I'm also happy for these improvements to be done on a separate PR. It would be great to have a new ticket for this work before closing the existing one/merging this PR, though.

Reply:
Great idea! I will add more logging to initialise.